### PR TITLE
`data:vamsipenmetsa7`

### PR DESCRIPTION
**Pull request by vamsipenmetsa7**
Thanks you eddie for awesome project

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3834"></a>

### CODE REVIEW

**Comment by vamsipenmetsa7** [comment]
Can someone approve this PR. Also please keep in mind that openapp link will work with out https: tag. So please merge my request

**Comment by vamsipenmetsa7** [comment]
added the required changes to the json file please review and merge my PR ASAP. Thanks in advance

File: data/vamsipenmetsa7.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  "type": "personal",
+  "bio": "🧐 Curious DevOps Dude | 20K+ smart minds 👀 my content on LinkedIn | 1.5+ million constant impressions 📈 | ♾ DevOps Community Builder | Azure ☁️ | love 🥑 | 🛫 From 🇮🇳, Lives in Europe 🇪🇺 | 💁🏻‍♂️ 𝗙𝗢𝗟𝗟𝗢𝗪",
+  "avatar": "https://github.com/vamsipenmetsa7.png",
+   "socials": [
+    { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/vamsi-penmetsa-392798119/" },
+    { "icon": "FaTwitter", "url": "https://twitter.com/vamsi_penmetsa" },
+    { "icon": "FaUsers", "url": "https://linkedin.openinapp.co/groups-13986647" },
+    {
+      "icon": "FaDollarSign",
+      "url": "https://github.com/sponsors/Pradumnasaraf"
```
Comment (Aadarsh805) [inline data/vamsipenmetsa7.json:13]: This should be your GitHub sponsors link, fix this please

File: data/vamsipenmetsa7.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    
+    "milestones": [
+   
+    {
+      "title": "Crossed 20K followers on LinkedIn",
+      "date": "2022",
+      "icon": "FaLinkedin",
+      "color": "purple",
+      "description": "Creating content on DevOps and connecting 20K followers milestone 🥳",
+      "url": "https://www.linkedin.com/in/pradumnasaraf"
```
Comment (Aadarsh805) [inline data/vamsipenmetsa7.json:52]: This LinkedIn URL is also not yours

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

**Comment by vamsipenmetsa7** [comment]
@Aadarsh805  I have updated the links now

File: data/vamsipenmetsa7.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/vamsi_penmetsa",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Subscribe to my Youtube channel",
+      "url": "https://www.youtube.com/channel/UCecon2EsKZyPxvjA4WEaGlA",
+      "icon": "FaYouTube"
```
Comment (Aadarsh805) [inline data/vamsipenmetsa7.json:26]: All good just this youtube icon, use `FaYoutube`

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: data/vamsipenmetsa7.json (hunk 2 above)
Comment (vamsipenmetsa7) [reply data/vamsipenmetsa7.json]: done

File: data/vamsipenmetsa7.json (hunk 3 above)
Comment (vamsipenmetsa7) [reply data/vamsipenmetsa7.json]: changed it, Thanks.

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**